pycytominer/operations/noise_removal.py

```diff
@@ -43,7 +43,7 @@ def noise_removal(
     """
     # Subset dataframe
     if samples != "all":
-        population_df.query(samples, inplace=True)
+        population_df = population_df.query(samples)
```

Comment:
Similar comment here: consider adding more detail through a code comment which describes what's happening here and why.